website/docs/usage/embeddings-transformers.md

```diff
@@ -351,9 +351,12 @@ factory = "transformer"
 max_batch_items = 4096
 
 [components.transformer.model]
-@architectures = "spacy-transformers.TransformerModel.v1"
+@architectures = "spacy-transformers.TransformerModel.v3"
 name = "bert-base-cased"
 tokenizer_config = {"use_fast": true}
+transformer_config = {"output_attentions": false}
```

Comment:
```suggestion
transformer_config = {"output_attentions": true}
```

Reply:
Wait here we need to ensure that the config mimicks the Python code above it. So either add the same non-default values to the Python code, or update the config to use the defaults. Either way, it's probably best to be explicit in both versions, so they can be matched visually easily by the reader of the docs.

Reply:
Ah, I think I got the sequence mixed up with the python vs. config blocks. Whatever you do, it doesn't make sense to say `{"output_attentions": false}` everywhere. Just leave the default as it is and use `{}` or nothing instead.